common/p_setup.cpp

```diff
@@ -1819,7 +1847,13 @@ CVAR_FUNC_IMPL(sv_timelimit)
 
 CVAR_FUNC_IMPL(sv_intermissionlimit)
 {
-	level.inttimeleft = (var < 1 ? DEFINTSECS : var);
+	if (sv_gametype == GM_COOP && var < 10) {
```

Comment:
Any particular reason for enforcing a 10 second minimum for co-op?

Reply:
Nevermind, I see this code was part of the previous implementation:
`int intlimit = (sv_intermissionlimit < 1 || sv_gametype == GM_COOP ? DEFINTSECS : sv_intermissionlimit);`